dao/src/main/java/org/tango/transport/HttpTransport.java

```diff
@@ -0,0 +1,60 @@
+package org.tango.transport;
+
+
+import java.io.IOException;
+import java.net.URI;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+import java.nio.charset.StandardCharsets;
+import java.time.Duration;
+
+import static java.net.http.HttpClient.Version.HTTP_2;
+
+/**
+ * @author Igor Khokhriakov <igor.khokhriakov@hzg.de>
+ * @since 20.02.2020
+ */
+public class HttpTransport implements Transport {
+    private HttpClient client;
+    private URI uri;
+
+    @Override
+    public boolean isConnected() {
+        return uri != null;
+    }
+
+    @Override
+    public void connect(String endpoint) throws IOException {
```

Comment:
Logging here? Also there probably should be a check about connecting, when connection already exists.